app/src/main/java/com/hedvig/app/feature/welcome/WelcomeDialog.kt

```diff
@@ -16,7 +16,7 @@ class WelcomeDialog : DismissablePager() {
 
     override val tracker: WelcomeTracker by inject()
     override val items: List<DismissablePagerPage> by lazy {
-        arguments!!.getParcelableArrayList<DismissablePagerPage>(ITEMS)
+        arguments!!.getParcelableArrayList<DismissablePagerPage>(ITEMS)!!
```

Comment:
Could be nice to do a ?: throw("custom error instead") Kotlin null pointer exception can be hard to decipher :)